wazo_confd/plugins/ingress_http/resource.py

```diff
@@ -28,7 +28,16 @@ def post(self):
 
     @required_acl('confd.ingresses.http.read')
     def get(self):
-        return super().get()
+        result = super().get()
```

Comment:
`Resource` is not really intended to use `super()` when you need a different logic
i.e. The base class has already all helpers to build your logic, so you can just use them explicitly without passing by `super` and the code will be much more readable